dlang.org.ddoc

```diff
@@ -155,6 +155,7 @@ DBLOG_LATEST_DATE2=
 DBLOG_LATEST_LINK2=https://dlang.org/blog
```

Comment:
Pedantically this should be `$(HTTPS dlang.org/blog)`. That way if the macro ever makes it in non-html documentation, there's no problem with it.

Reply:
Not related to this PR and it's only used for the front page. The reason it's defined here, is that DAutoTest doesn't error with undefined macros as these are dynamically injected.